client/my-sites/checkout/checkout-thank-you/jetpack-instruction-list.tsx

```diff
@@ -0,0 +1,20 @@
+import { FC, ReactNode } from 'react';
+
+export interface JetpackInstructionListProps {
+	items: ReactNode[];
+}
+
+const JetpackInstructionList: FC< JetpackInstructionListProps > = ( { items } ) => {
+	return (
+		<ol className="jetpack-instructions__list">
+			{ items.map( ( item, index ) => (
+				<li className="jetpack-instructions__list-item">
```

Comment:
When rendering a list/array of elements like this, every element in the list [must have a `key` prop](https://reactjs.org/docs/lists-and-keys.html#keys) passed to it. Otherwise you will see console warnings about it.

```suggestion
				<li className="jetpack-instructions__list-item" key={ index }>
```

Reply:
Using `index` to remove the React warning is generally worse than leaving the warning in-place, because it will hide the bugs that can be caused if the items ever change.

Reply:
It's not bad for the lists in which the items won't change.